src/player.cpp

```diff
@@ -430,15 +430,8 @@ uint32_t Player::getClientIcons() const
 		icons &= ~ICON_SWORDS;
 	}
 
-	// Game client crash with more than 10 icons
-	// so let's prevent that from happening.
-	std::bitset<20> icon_bitset(static_cast<uint64_t>(icons));
-	for (size_t pos = 0, bits_set = icon_bitset.count(); bits_set >= 10; ++pos) {
-		if (icon_bitset[pos]) {
-			icon_bitset.reset(pos);
-			--bits_set;
-		}
-	}
+	std::bitset<31> icon_bitset(static_cast<uint64_t>(icons));
```

Comment:
I believe that this line is not necessary anymore and we can just `return icons;` now.